lib/repository/local/sqlite/dictionary_items/dictionary_items_local_repository.dart

```diff
@@ -314,3 +391,12 @@ class DictionaryItemsLocalRepository extends DictionaryItemsLocalRepositoryAPI {
   DictionaryGroup _getGroup(int groupNumber) =>
       DictionaryGroup.values.firstWhere((DictionaryGroup group) => group.groupNumber == groupNumber);
 }
+
+extension HiraKataConvertion on String {
+  String toHiragana() {
+    return replaceAllMapped(RegExp('[ァ-ヴ]'), (Match match) => String.fromCharCode(match.group(0)!.codeUnitAt(0) - 0x60));
+  }
+  String toKatakana() {
+    return replaceAllMapped(RegExp('[ぁ-ゔ]'), (Match match) => String.fromCharCode(match.group(0)!.codeUnitAt(0) + 0x60));
+  }
+}
```

Comment:
- Utilに追加する
- 名前も命名規則っぽいのあったほうがいいからHogeExtensionで統一しよう
```suggestion
extension StringExtension on String {
  String toHiragana() {
    return replaceAllMapped(RegExp('[ァ-ヴ]'), (Match match) => String.fromCharCode(match.group(0)!.codeUnitAt(0) - 0x60));
  }
  
  String toKatakana() {
    return replaceAllMapped(RegExp('[ぁ-ゔ]'), (Match match) => String.fromCharCode(match.group(0)!.codeUnitAt(0) + 0x60));
  }
}
```